apps/src/assetManagement/assetPrefix.js

```diff
@@ -54,8 +55,11 @@ export function fixPath(filename) {
   // Rewrite urls to pass through our media proxy. Unless of course we are in an
   // exported app, in which case our media proxy won't be good for anything
   // anyway.
-  if (ABSOLUTE_REGEXP.test(filename) && window.location.protocol !== 'file:') {
-    if (ABSOLUTE_CDO_CURRICULUM_REGEXP.test(filename)) {
+  if (ABSOLUTE_REGEXP.test(filename) && !window.location.protocol !== 'file:') {
+    if (
```

Comment:
Just curious why you added `!` in front of `window.location.protocol`. Interestingly, it looks like it works either way, but unsure why.
Also, it looks like '//images.code.org...' still works even though 'https://images.code.org/' is used as the parameter for the `RegEx`. Do you mind explaining why? Thanks!

Reply:
I did not mean to modify the check on `window.location.protocol` -- thank you!

`//images.code.org` was already being handled [here](https://github.com/code-dot-org/code-dot-org/pull/50828/files#diff-06ff5e2d2890426aceac23d1891a79613e9e82754e5c5f45e5e7333404023662R95). Because it's not an absolute path, it misses this condition entirely.

Reply:
Thanks! Is the check for window.location.protocol necessary? If so, maybe add a unit test when the value is assigned 'file:'. If not, then remove?

Reply:
I think it's used to test if we're in an exported app and plan to leave it in place. I did not add a unit test for this case because I don't think it's clean or advisable to mock or modify the `window` for the test -- I might be wrong!